kaolin/datasets/shapenet.py

```diff
@@ -113,7 +113,7 @@ def download_shapenet_class(syn: str, shapenet_location: str, download: bool):
     NotImplemented
 
 
-def download_images(shapenet_root: str):
+def download_images(shapenet_location: str):
```

Comment:
Shouldn't we just get ride of this ? (and `download` argument on the __init__)

Reply:
Agreed.